src/core/viz/expressions/torque.js

```diff
@@ -186,14 +194,13 @@ export class Torque extends BaseExpression {
      * @instance
      * @name getSimTime
      */
-    
     getSimTime() {
         const progress = this.progress.eval(); //from 0 to 1
         const min = this.input.min.eval();
         const max = this.input.max.eval();
 
         if (!(min instanceof Date)) {
-            return progress * (max - min) + min;
+            return new Date(progress * (max - min) + min);
```

Comment:
This seems wrong to me. If `min` is not an instance of `Date` that should be because the input is numeric. Therefore, the output of `getSimTime` should be a number, not a Date. In particular, it is not possible to do this cast because input could be in seconds, weeks, or years, but we don't know which unit